agent/util-scripts/pbench-sysinfo-dump

```diff
@@ -86,27 +86,36 @@ function collect_block {
 
 function collect_sos {
 	read sos_ver sos_ver_minor sos_ver_subminor <<< `rpm -q sos | awk -F- '{print $2}' | awk -F. '{print $1" "$2" "$3}'`
-	if [[ "$sos_ver" -ge 3 ]]; then
-		_quiet="--quiet"
-		if [[ "$sos_ver_minor" -ge 6 || ( "$sos_ver_minor" -eq 5 && ! -z "$sos_ver_subminor" ) ]]; then
-			_modules="date host release"
-		else
-			_modules="general lsbrelease"
-		fi
-		_modules="${_modules} block processor"
-		if [[ "$sos_ver_minor" -ge 4 ]]; then
-			# ref for plugin collectd: https://github.com/sosreport/sos/pull/866
-			_modules="${_modules} collectd"
-		fi
-		if [[ "$sos_ver_minor" -eq 0 ]]; then
-			# tuned plug-in in sosreport v3.1+
-			_modules="${_modules} tuned"
-		fi
-	else
+	if [[ "${sos_ver}" -lt 3 ]]; then
 		_quiet=""
+	else
+		_quiet="--quiet"
+	fi
+	if [[ ${sos_ver} -lt 3 || ( ${sos_ver} -eq 3 && ( "${sos_ver_minor}" -lt 5 || ( "${sos_ver_minor} -eq 5 && -z "${sos_ver_subminor}" ) ) ) ]]; then
+		# Pre-v3.5.1+
```

Comment:
OK, wow; so it was actually wrong before, not just "not future-proofed".

Reply:
Yes, thanks for pointing that out, we needed this fix.